scripts/styleguide-build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
```

Comment:
If you want to be able to run this from a directory other than the project root, adding this simple one-liner at the top should do the trick:

```shell
cd "$(dirname "$0")/.."
```